README.md

```diff
@@ -6,43 +6,49 @@ Form container is a lightweight React form container with validation (written in
 It allows you to use both HTML5 form validation and custom validation functions.
 
 It provides your child form with 2 objects of props:
-- `form` - all data on your form values, states and errors
```

Comment:
I'm not sure why there's so many formatting changes in this file and others. I ran `pretty-quick` so guessing that's why?